shell/update-counts.sh

```diff
@@ -28,20 +22,19 @@ update_counts() {
     local output=$(FOUNDRY_PROFILE=benchmark forge t --mt "test_Estimate${test_name}" -vv)
     echo -e "\nParsing output for $test_name..."
 
-    # Define a table with headers
+    # Define a table with headers. This table is not put in the Solidity script file.
     local table="Category,Chain ID,New Max Count"
 
     # Parse the output to extract counts and chain IDs
     while IFS= read -r line; do
         local count=$(echo $line | awk '{print $2}')
         local chain_id=$(echo $line | awk '{print $8}')
-        local formatted_chain_id=$(format_chain_id $chain_id)
 
         # Add the data to the table
-        table+="\n$map_name,$formatted_chain_id,$count"
+        table+="\n$map_name,$chain_id,$count"
 
         # Update the map for each chain ID using sd
-        sd "$map_name\[$formatted_chain_id\] = [0-9_]+;" "$map_name[$formatted_chain_id] = $count;" $BASE_SCRIPT
+        sd "$map_name\[$chain_id\] = [0-9_]+;" "$map_name[$chain_id] = $count;" $BASE_SCRIPT
```

Comment:
do we need the underscore in this regex?

`[0-9_]+`

Reply:
you are correct, we no longer need underscore since we have removed `format_chain_ids`